odoo/addons/base/models/ir_filters.py

```diff
@@ -140,16 +140,9 @@ def create_or_replace(self, vals):
 
         return self.create(vals)
 
-    _sql_constraints = [
-        # Partial constraint, complemented by unique index (see below). Still
-        # useful to keep because it provides a proper error message when a
-        # violation occurs, as it shares the same prefix as the unique index.
-        ('name_model_uid_unique', 'unique (name, model_id, user_id, action_id)', 'Filter names must be unique'),
-    ]
```

Comment:
Instead of dropping this constraint, could we instead remove the `translate` flag on ir_filter.name? That's a very old legacy behavior that wasn't well thought out. 

There are no built-in ir.filters that need to be translated, and most filters are either personal or per-team, where translations are not useful.

The unique constraint is great to prevent duplicate entries in search views, where you can't tell a filter from another one (most people don't know about "Manage Filters" etc.)

Reply:
Making that field non-translated would make more sense.